adafruit_bno055.py

```diff
@@ -392,7 +392,9 @@ def _write_register(self, register, data):  # pylint: disable=arguments-differ
         if not isinstance(data, bytes):
             data = bytes([data])
         self._uart.write(bytes([0xAA, 0x00, register, len(data)]) + data)
-        time.sleep(0.1)
+        now = time.time()
```

Comment:
Use `time.monotonic` instead of `time.time` because it'll work on all CP boards that way. `time.time` needs to know the calendar time while monotonic is relative to start up.

Reply:
Will do